metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -107,11 +107,17 @@ class PostgresConfig(BasicSQLAlchemyConfig):
 
     database_pattern: AllowDenyPattern = Field(
         default=AllowDenyPattern.allow_all(),
-        description="Regex patterns for databases to filter in ingestion.",
+        description=(
+            "Regex patterns for databases to filter in ingestion. "
+            "Note: this is not used if `database` or `sqlalchemy_uri` are provided."
+        ),
     )
-    database: Optional[str] = Field(
-        default=None,
-        description="database (catalog). If set to Null, all databases will be considered for ingestion.",
+    init_database: Optional[str] = Field(
+        default="postgres",
+        description=(
+            "Initial database used to query for the list of databases, when ingesting multiple databases. "
+            "Note: this is overwritten by the `database` and `sqlalchemy_uri` config params."
```

Comment:
```suggestion
            "Note: this is not used if `database` or `sqlalchemy_uri` are provided."
```

Reply:
"overwritten" seems more like code related thing and need not be present in doc.

Reply:
I think I meant overridden lol, also a cs term but I think is applicable here

Reply:
Or perhaps I can just make it so you can't specify both